scripts/bench_python_cpp_algos.py

```diff
@@ -452,9 +518,7 @@ def load_graph(graph_path, edge_properties=None):
     )
     parser.add_argument("--thread-spin", default=False, action="store_true", help="Busy wait for work in thread pool.")
 
-    parser.add_argument("--save-json", default=False, help="Saving the benchmarking information as a JSON file.")
-
-    parser.add_argument("--save-dir", default="./", help="Path to the save directory (default: %(default)s)")
+    parser.add_argument("--json-output", default="", help="Path to the save directory (default: %(default)s)")
```

Comment:
"Path to the save directory" is not semantically meaningful. :-) I think you mean "Path at which to save performance data in JSON" or something like that. And no reason to include the default value since it doesn't really make sense.